### ISSUE TITLE
Evaluate Twine Usage

### ISSUE THREAD

**Issue by ColmDC** [DigitalCommons/twine-together.coop#1]
Identify how much Twine is currently being used.

We have no idea how many active users there are of Twine. This can be a once-off exercise, to get a baseline, rather than implementing a usage tracking system. I'm also not clear what the relevant acitivities to measure are, so we may need to start with that. 

Identify what aspects of Twine usage we should measure to give a sense of its overall activity.  

Candidate questions:

- How many active accounts are there?
- How frequently are the back ends of those accounts used?
- How many member/visitor 'visits/activities' are logged on these accounts?

**Comment by ColmDC** [comment]
What do you think of this exercise @King-Mob ?

**Comment by King-Mob** [comment]
@ColmDC I think I could do a once-off look at the above in about an hour and get relevant information. I'm also adding the twine admin backend and the visitor site to our analytics service umami, so that'll give us more detailed breakdowns in a more convenient way

**Comment by ColmDC** [comment]
Great. Could you do this and write up whatever you find for the end of the month?

**Comment by King-Mob** [comment]
@ColmDC Yep. Lynne is also good at turning database information into a general picture of a service so I'd be happy to give her access to the db

**Comment by ColmDC** [comment]
Let's tag her in and get her thoughts @lin-d-hop
